.travis/runpytest.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Wrapper around pytest. First argument is a path to environment python, the
```

Comment:
Can you please add the reason why we need this wrapper to the comment?

Reply:
@i386x

Reply:
Done in #144.